workspace/cmd/api1/api1.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"log"
+	"net/http"
+
+	apiv1 "github.com/bufbuild/buf-examples/workspace/gen/proto/go/api/v1"
+	"github.com/bufbuild/buf-examples/workspace/gen/proto/go/api/v1/apiv1connect"
+	"github.com/bufbuild/connect-go"
+	"golang.org/x/net/http2"
+	"golang.org/x/net/http2/h2c"
+	emptypb "google.golang.org/protobuf/types/known/emptypb"
+)
+
+func main() {
+	const addr = "127.0.0.1:8080"
```

Comment:
nit: Maybe to make this a little less strange looking it could be made into a flag?
```go
addr := flag.String("listen", "127.0.0.1:8080", "address to listen on")
flag.Parse()
```
The only other change is that since it will be `*string` it would need to be dereferenced in the two places it's referenced.
Same idea for the other one.
This is probably a little overkill for the demo, but it probably would look a little cleaner.